source/module_elecstate/module_charge/charge_mixing.h

```diff
@@ -45,18 +46,22 @@ class Charge_Mixing
      *
      */
     void Kerker_screen_recip(std::complex<double>* rhog);
+    void Kerker_screen_recip_new(std::complex<double>* rhog);
 
     /**
      * @brief Kerker screen method for real space
      *
      */
     void Kerker_screen_real(double* rho);
+    void Kerker_screen_real_test(double* rho);
 
     /**
      * @brief Inner product of two complex vectors
      *
      */
     double inner_product_recip(std::complex<double>* rho1, std::complex<double>* rho2);
+    double inner_product_recip_new1(std::complex<double>* rho1, std::complex<double>* rho2);
+    double inner_product_recip_new2(std::complex<double>* rho1, std::complex<double>* rho2);
```

Comment:
These new functions should add Unit Tests.

Reply:
Done.